src/apis/YouTube.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+const KEY = 'AIzaSyACcAkV07ihRRybdSU6WNdOJF__uOQFEEY';
```

Comment:
It is not a good idea to upload your API Keys to github as it is publicly available and bots and/or people might steal it.

Look into using `.env` package to use environment variables to have variables like the apikey locally and on the deployed server but not on github

[Reference](https://create-react-app.dev/docs/adding-custom-environment-variables/)